app/views/foods/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Results</h1>
+
+<p>Total Items: <%= @food_results.count %> </p>
```

Comment:
Ideally this would be a count of all foods containing sweet potatoes (which is something like 30,000). Since your API request is imposing a page size limit of 10 this will always be 10. I would probably remove the limit of 10 on the API request in the service to get the count and then take the first 10 items and assign them to POROs for the view.